packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -74,11 +78,21 @@ export class WaferMap extends FoundationElement {
     @attr({ attribute: 'color-scale-mode' })
     public colorScaleMode: WaferMapColorScaleMode = WaferMapColorScaleMode.linear;
 
+    /**
+     * @internal
+     */
+    public workerOne!: Remote<MatrixRenderer>;
+
     /**
      * @internal
      */
     public readonly canvas!: HTMLCanvasElement;
 
+    /**
+     * @internal
+     */
```

Comment:
Why are we exposing two different canvas elements on the API surface (particularly when they are the same type)?